packages/oidc/src/token.ts

```diff
@@ -0,0 +1,226 @@
+interface TokenPayload {
+  sub: string;
+  name: string;
+  exp: number;
+}
+
+function isNodeEnv(): boolean {
+  return (
+    typeof process !== 'undefined' &&
+    process.versions != null &&
+    process.versions.node != null
+  );
+}
+
+function isBrowserEnv(): boolean {
+  return typeof window !== 'undefined';
+}
+
+async function getVercelDataDir(): Promise<string | null> {
+  if (isBrowserEnv()) {
+    return '';
+  }
```

Comment:
I think this could be removed, because I cannot see where this is called without first passing through such a check?